src/PAModel/Utility/Utilities.cs

```diff
@@ -148,16 +148,19 @@ public static JsonElement ToJson(this ZipArchiveEntry e)
         // returns "bar\hi.txt"
 
         // Net Core 2.1 provides a Path.GetRelativePath api
-        // but since we target netstandard, we can convert to URIs and make the relative path from that
+        // but since we target netstandard 2.0, we can convert to URIs and make the relative path from that
         // see https://stackoverflow.com/questions/275689/how-to-get-relative-path-from-absolute-path
-        public static string GetRelativePath(string full, string basePath)
+        // For reference, see Core's impl at: https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/IO/Path.cs#L861
+        public static string GetRelativePath(string fullPathFile, string basePathDirectory)
```

Comment:
NIT - I would suggest making a change to the signature here such that the `basePathDirectory` swaps positions with the `fullPathFile` so that it conforms with the `Path.GetRelativePath` method that we're mimicking here.  I would also suggest renaming the parameters to be the same as well.